### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by lukman155**
# **Set up a "Hello Microverse" project**
## Learning Objectives

- Create branches with different base branches.
- Understand the concept of version control.
- Follow GitHub flow.
- Open pull requests in GitHub.
- Write descriptive, easy-to-understand commit messages.
- Write short, detailed, easy-to-understand descriptions of pull requests.
- Write a descriptive and easy-to-understand README in business English for every project submitted.
- Demonstrate an ability to submit a project for code review.

## Description
In this project, I set up a "Hello world" repository. No complex coding was required for this exercise.
my goal here is to master all of the tools and best practices I learned about in previous steps.

##General requirements
- I made sure that there are no linter errors.
- I made sure that I used the correct GitHub flow.
- I made sure that I documented your work.
## Requirements

- I documented my work [in a professional way.
- I created a new GitHub repo.
- I created a new branch according to the GitHub flow rules.
- I Added a .gitignore file.
- I Set up a linter for HTML and CSS and made sure that any files that are not relevant are ignored by git.
- I added a descriptive README file to my project - I modified [this template](https://github.com/microverseinc/readme-template) for my project.
- I Added 3 files (each one in a separate commit):
- A .html file that includes a header with the text "Hello Microverse!". I Used h1 tag for that.
- A .css file that includes one class with styles for my header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".
- I made sure that the last file is actually ignored by git and is not present in my GitHub repository.
- I opened a pull request.
# 🤗

### CODE REVIEW

File: .gitignore.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+test.md
+app_screenshot
```
Comment (andgarzonmal) [inline .gitignore.txt]: - [x] Kindly unse a .gitignore file, in this case this file is a text file, please remove the .txt from your .gitignore so git knows wich files to ignore.

File: .gitignore.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+test.md
```
Comment (andgarzonmal) [inline .gitignore.txt]: - [ ]  Kindly remember to add a test.md file with the text "This file should be ignored by git" to your repository (This is the file you should ignore on .gitignore file `test.md/` )

File: .hintrc.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```
Comment (andgarzonmal) [inline .hintrc.txt]: [OPTIONAL] Kyndly remove the .txt extension from your .hintrc.txt file (it should be just  `.hintrc` ) you are adding a text file istead of the instructions of the linter.

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @lukman155    ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by roshan-bajgain — APPROVED** [review]
Hi @lukman155, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**